### PR TITLE
Delete some orphaned BuildFiles

### PR DESCRIPTION
**Pull request by guitargeek**
#### PR description:

Delete 3 BuildFiles which were actually not building any sources.

#### PR validation:

CMSSW compiles.

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

No backport intended.

### CODE REVIEW

**Comment by pohsun** [comment]
please test

File: FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc [hunk 1]
```diff
@@ -16,18 +16,124 @@ Description: [one line class summary]
...
+
+class CTPPSFastTrackingProducer : public edm::stream::EDProducer<> {
+public:
+  explicit CTPPSFastTrackingProducer(const edm::ParameterSet&);
+  ~CTPPSFastTrackingProducer() override;
+
+private:
+  void beginStream(edm::StreamID) override;
+  void produce(edm::Event&, const edm::EventSetup&) override;
+  void endStream() override;
```
Comment (makortel) [inline FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc:55]: This probably goes beyond this PR, but `beginStream()` and `endStream()` are empty and thus could be removed.

File: FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc [hunk 2]
```diff
@@ -16,18 +16,124 @@ Description: [one line class summary]
...
+  //this function will only be called once per event
+  virtual void beginEvent(edm::Event& event, const edm::EventSetup& eventSetup);
+  virtual void endEvent(edm::Event& event, const edm::EventSetup& eventSetup);
```
Comment (makortel) [inline FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc:58]: Again probably going beyond this PR, but what is the purpose of these functions? As far as I can tell, they are not called.

File: Utilities/ReleaseScripts/scripts/git-publish [hunk 3]
```diff
@@ -455,7 +455,7 @@ CMSSW_CATEGORIES={"Operations": ["Configuration/Applications",
   "FastSimulation/Configuration", "FastSimulation/EgammaElectronAlgos",
   "FastSimulation/Event", "FastSimulation/EventProducer",
   "FastSimulation/ForwardDetectors", "FastSimulation/HighLevelTrigger",
-  "FastSimulation/L1CaloTriggerProducer", "FastSimulation/MaterialEffects",
+  "FastSimulation/MaterialEffects",
```
Comment (makortel) [inline Utilities/ReleaseScripts/scripts/git-publish:458]: @smuzaffar Could you confirm this is OK given that `FastSimulation/L1CaloTriggerProducer` package is removed?

File: FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc (hunk 2 above)
Comment (guitargeek) [reply FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc]: You are right, thanks! I removed them.

File: FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc (hunk 1 above)
Comment (guitargeek) [reply FastSimulation/CTPPSFastTrackingProducer/plugins/CTPPSFastTrackingProducer.cc]: Sure!

**Comment by guitargeek** [comment]
Hi @makortel, I addressed your comments and removed one more BuildFile. Could you lanch the tests again? Thanks!

File: Utilities/ReleaseScripts/scripts/git-publish (hunk 3 above)
Comment (smuzaffar) [reply Utilities/ReleaseScripts/scripts/git-publish:458]: Yes this is fine

**Comment by smuzaffar** [comment]
Please test

**Comment by makortel** [comment]
+core

**Comment by pohsun** [comment]
+1

**Comment by civanch** [comment]
@jan-kaspar , @fabferro , are you OK with this PR?

**Comment by fabferro** [comment]
> 
> 
> @jan-kaspar , @fabferro , are you OK with this PR?

I have nothing against but I would like the confirmation of @mundim that last modified these files within PPS group. @mundim: do you agree with this PR?

**Comment by silviodonato** [comment]
Do you have any objections @ggovi ?

**Comment by mundim** [comment]
> > @jan-kaspar , @fabferro , are you OK with this PR?
> 
> I have nothing against but I would like the confirmation of @mundim that last modified these files within PPS group. @mundim: do you agree with this PR?

I was not sure about FastSimulation/CTPPSFastTrackingProducer, if this would cause any problem to the PPS fast simulation, but I just found out that (likely) no one is using it. So, I guess you can go on.

**Comment by civanch** [comment]
+1

**Comment by ggovi** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**